File: Baekjoon/step by step/07_2d_array/2563.py
# 2563: 색종이

N = int(input())
paper = [[0 for j in range(100)] for i in range(100)]

for _ in range(N):
  x, y = list(map(int, input().split()))

  for i in range(10):
    for j in range(10):
      paper[i+x][j+y] = 1

area = 0

for i in range(100):
  for j in range(100):
    area += paper[i][j]

print(area)

File: Baekjoon/step by step/07_2d_array/2563.cpp
// 2563: 색종이

#include <iostream>

int main() {
  int N, area = 0;
  int paper[100][100] = {0};
  
  std::cin >> N;

  while (N--) {
    int x, y;
    std::cin >> x >> y;

    for (int i=0; i<10; i++) {
      for (int j=0; j<10; j++) {
        paper[i+x][j+y] = 1;
      }
    }
  }

  for (int i=0; i<100; i++) {
    for (int j=0; j<100; j++) {
      area += paper[i][j];
    }
  }

  std::cout << area;

  return 0;
}